docker-compose.yml

```diff
@@ -13,16 +13,13 @@ services:
 
   firefly-app:
     image: firefly-iii
-    build:
-      context: .
+    build: .
     environment:
-      - INIT_DATABASE=yes
```

Comment:
Agree, making it a separate task sounds more reasonable than using the environment variable.